README.md

```diff
@@ -165,6 +165,24 @@ in the Kube cluster. You will need to create a ClusterRoleBinding with proper
 security settings. For information about the role bindings,
 take a look at the info [here](https://kubernetes.io/docs/admin/authorization/rbac/).
 
+## Kubernetes Host Linux Versions
+
+Unfortunitaly when Deploying OpenWhisk on Kubernetes it currently mounts some
+of the host OS files for the Invoker process and needs to make some assumptions.
+Because of this, some failures are know to happen on certain Linux versions,
```

Comment:
```markdown
Because of this, some failures are know to happen on certain Linux versions,
```

Should be:

```markdown
Because of this, some failures are known to happen on certain Linux versions,
```